README.md

```diff
@@ -243,6 +242,56 @@ In the strict mode, `async` functions are checked such that they only raise
 effects on forward declarations, callbacks and methods using
 `{.raises: [CatchableError].}` (or more strict) annotations.
 
+### Cancellation support
+
+Any running `Future` can be cancelled.
+
+```nim
+proc c1 {.async.} =
+  echo "Before sleep"
+  try:
+    await sleepAsync(10.minutes)
+    echo "After sleep" # never reached
+  except CancelledError as exc:
+    echo "We got cancelled!"
+    raise exc
+
+proc c2 {.async.} =
+  await c1()
+  echo "Never reached, since the CancelledError bubble up"
+
+let work = c2()
+waitFor(work.cancelAndWait())
+```
+
+The cancellation will travel down the call stack, and throw a `CancelledError`
```

Comment:
There is no "stack". There is sequence of Futures, but it is definitely not a call stack.

Reply:
Just like the "call stack" is not actually a stack of calls
It's an implementation details whether it's a stack, a sequence or a linked list